fe/fe-core/src/main/java/com/starrocks/catalog/HiveTable.java

```diff
@@ -176,12 +176,13 @@ public String getCatalogName() {
         return catalogName == null ? getResourceMappingCatalogName(resourceName, "hive") : catalogName;
     }
 
+    @Override
     public String getDbName() {
         return hiveDbName;
     }
 
     @Override
-    public String getTableName() {
+    public String getName() {
```

Comment:
missing iceberg table?

Reply:
@stephen-shelby  you mean

```    public String getResourceName() {
        return resourceName;
    }

    public String getRemoteDbName() {
        return remoteDbName;
    }

    public String getRemoteTableName() {
        return remoteTableName;
    }

```
Changed to getDbName and getName?

Reply:
@stephen-shelby can you take a look at my fix?